.github/ISSUE_TEMPLATE/test-plan.md

```diff
@@ -0,0 +1,62 @@
+## Test plan
+
+## Feature
+
+Name:
+
+Description:
+
+Group: CX / Infra / MDM / Website
```

Comment:
consider linking to original issue instead, where this can be quickly referenced